src/main/java/ch/njol/skript/expressions/ExprDustedStage.java

```diff
@@ -62,6 +62,7 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 		if (!isMax && mode == ChangeMode.SET) {
 			return CollectionUtils.array(Integer.class);
 		}
+		error("Attempting to modify the max dusted stage is not supported.");
 		return null;
 	}
```

Comment:
This can cause incorrect errors, what if someone does `add 1 to max dusted stage of {_block}`?